posit_team_applications_faq.md

```diff
@@ -12,6 +12,10 @@ This document aims to answer frequently asked questions from users in relation t
 
 ### Installing Packages
 
+#### Where are all my packages?
+
+Unlike the old RStudio Server Pro environment, there are no pre-installed packages in Posit Workbench.  This gives you full control over what packages, and versions of packages, you want to use, rather than those dictated by the administrator.
```

Comment:
```suggestion
Unlike the old RStudio Server Pro environment, there are no pre-installed packages in Posit Workbench.  This gives you full control over what packages, and versions of packages, you want to use, rather than those dictated by the administrator. You can install any needed packages with `install.packages("<package name>")` or using the GUI. 
```